actions/pacstrap_action.go

```diff
@@ -0,0 +1,129 @@
+/*
+Pacstrap Action
+
+Construct the target rootfs with pacstrap tool.
+
+	# Yaml syntax:
+	- action: pacstrap
+	  config: <in-tree pacman.conf file>
+	  mirror: <in-tree mirrorlist file>
+
+Mandatory properties:
+
+  - config -- the pacman.conf file which will be used through the process
+  - mirror -- the mirrorlist file which will be used through the process
+*/
+package actions
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+
+	"github.com/go-debos/debos"
+	"github.com/go-debos/fakemachine"
+)
+
+type PacstrapAction struct {
+	debos.BaseAction `yaml:",inline"`
+	Config           string `yaml:"config"`
+	Mirror           string `yaml:"mirror"`
+}
+
+func (d *PacstrapAction) listOptionFiles(context *debos.DebosContext) ([]string, error) {
+	files := []string{}
+
+	if d.Config == "" {
+		return nil, fmt.Errorf("No config file set, aborting.")
+	}
+	d.Config = debos.CleanPathAt(d.Config, context.RecipeDir)
+	files = append(files, d.Config)
+
+	if d.Mirror == "" {
+		return nil, fmt.Errorf("No mirror file set, aborting.")
+	}
+	d.Mirror = debos.CleanPathAt(d.Mirror, context.RecipeDir)
+	files = append(files, d.Mirror)
+
+	return files, nil
+}
+
+func (d *PacstrapAction) Verify(context *debos.DebosContext) error {
```

Comment:
Should we check `context.Architecture==amd64` too ?

Reply:
As per the CPU architecture discussion above, the check should not exist.